docs/swagger.yaml

```diff
@@ -333,7 +333,7 @@ paths:
         This endpoint is for migrating existing MobilePay PoS QR codes from the current solution that will end its lifetime. It is meant for merchants that have printed QR codes and want them to stay functional for the new product offering that will replace the now deprecated solution.
 
         This endpoint will not create a new QR code but rather map the provided beaconId with the Merchant-Serial-Number, to make sure the already printed QR code can be re-used.
-        When the QR code is scanned by MobilePay users, it will result in a callback being sent to the merchant if the merchant has registered a [webhook](https://developer.vippsmobilepay.com/docs/APIs/webhooks-api) for the user.checkin.v1 event.
+        When the QR code is scanned by MobilePay users, it will result in a callback being sent to the merchant if the merchant has registered a [webhook](https://developer.vippsmobilepay.com/docs/APIs/webhooks-api) for the user.checked-in.v1 event.
```

Comment:
```suggestion
        When the QR code is scanned by MobilePay users, it will result in a callback being sent to the merchant if the merchant has registered a [webhook](https://developer.vippsmobilepay.com/docs/APIs/webhooks-api) for the `user.checked-in.v1` event.
```